tests/fixtures/theano/__init__.py

```diff
@@ -0,0 +1,4 @@
+from tests.fixtures.theano.batch_polopt_instrumented import \
+        InstrumentedBatchPolopt
+from tests.fixtures.theano.npo_instrumented import InstrumentedNPO
```

Comment:
Package names should be `instrumented_batchpolopt`, `instrumented_npo` and `instrumented_trpo` to match class names.